x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/playground/playground.tsx

```diff
@@ -15,10 +17,18 @@ import { KibanaLogic } from '../../../shared/kibana';
 import { EnterpriseSearchContentPageTemplate } from '../layout/page_template';
 
 export const Playground: React.FC = () => {
+  const [searchParams] = useSearchParams();
+  const index: string | null = searchParams.has('defaultIndexName')
```

Comment:
nitpick: indices as a parameter name

Reply:
It started with `index names` however after discussing with Yan changed it to `defaultIndexName`. The reason is once they are in Playground and may select other index or add more index, we are not planning to re-populate the url. So `url` indices might provide a confusion as why only one index name is showing up, not the others.

Reply:
thats fair. I dislike the camelcased for search params. `default-index=<index>` then?